lms/static/sass/base/_variables.scss

```diff
@@ -144,8 +210,8 @@ $linkedin-blue: #0077B5;
 // ====================
 
 // TODO: both blue and yellow variables differ from CMS rgb value, need to confirm change to CMS variable is ok in current platform uses before switching.
-$blue: rgb(0, 120, 176);
-$yellow: rgb(255, 252, 221);
+//$blue: rgb(0, 120, 176);
+//$yellow: rgb(255, 252, 221);
```

Comment:
for blue we'll want to keep the LMS blue instead of Studio, since that has been reviewed for a11y contrast. In fact, given your change above we should be able to remove this duplicated blue line. 

As I noted above, I will double check the yellow question with the team to get thoughts on that.